discovery-infra/test_infra/utils/libvirt_utils.py

```diff
@@ -0,0 +1,183 @@
+import json
+import logging
+
+import libvirt
+import xml.dom.minidom as md
+
+import waiting
+
+from logger import log
+
+from .. import consts
```

Comment:
when we will be able to remove this ? why it can't be "from test_infra import consts"?